components/MDX/Calendar/UpcomingMeetings.tsx

```diff
@@ -9,9 +9,9 @@ import type { CalendarEvent } from '@/types';
 
 import styles from './calendar.module.css';
 
-type GrouppedEntries = Record<string, Array<CalendarEvent>>;
+type GroupedEntries = Record<string, Array<CalendarEvent>>;
 
-const UpcomingEvents: FC = async () => {
+const UpcomingMeetings: FC = async () => {
```

Comment:
events is too loaded a term, as it is really pulling meeting data from the calendar, not `events` data from the blog